v4-client-py-v2/documentation/network_setup.md

```diff
@@ -1,52 +1,99 @@
 ### Networks
 
-> **See [network resources](https://docs.dydx.exchange/infrastructure_providers-network/resources#networks-repositories) to find publicly available endpoints**
+This guide explains how to connect to different dYdX networks using the Python SDK.
+
+#### Finding Network Endpoints
+
+> **Important:** For the most up-to-date list of publicly available endpoints, refer to our [network resources documentation](https://docs.dydx.exchange/infrastructure_providers-network/resources#networks-repositories).
+
+#### Connecting to Mainnet
+
+To connect to the mainnet, use the `make_mainnet` function:
 
-To connect to the mainnet you can use `make_mainnet` function:
 ```python
 from dydx_v4_client.network import make_mainnet
 
-
 NETWORK = make_mainnet(
-    node_url=NODE_URL, 
-    rest_indexer=REST_URL, 
-    websocket_indexer=WEBSOCKET_URL
+    node_url="dydx-ops-grpc.kingnodes.com:443",  # No 'https://' prefix
+    rest_indexer="https://indexer.v4.dydx.exchange",
```

Comment:
@Christopher-Li are we okay with suggesting indexer endpoints via docs? if so this looks like a pretty solid upgrade from our current documentation

Reply:
@samtin0x i'd prefer if we keep these listed as variable names and instead link to https://docs.dydx.exchange/infrastructure_providers-network/resources#indexer-endpoints.

This will make the docs more durable to changes in indexer endpoints and also let  developers know that they can set up their own indexer or node urls

Reply:
Yes, both approaches work. As long as users/devs have a simple example they can copy paste and get quickstarted, as that has been the main blocker here, hence the PR